### PR TITLE
Upgraded to Laravel 6

### PR DESCRIPTION
**Pull request by BoGnY**
* Laravel 6.0, 6.1, 6.2 support
* Updated dependencies
* Added italian language
* Some style fix

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -17,20 +17,9 @@
...
-        "illuminate/events": "^5.8",
-        "illuminate/database": "^5.8",
-        "illuminate/session": "^5.8",
-        "illuminate/routing": "^5.8",
-        "illuminate/http": "^5.8",
-        "illuminate/contracts": "^5.8",
-        "illuminate/view": "^5.8",
-        "illuminate/translation": "^5.8"
+        "php" : ">=7.2",
+        "laravel/framework": "^6.0",
```
Comment (lahaxearnaud) [inline composer.json:21]: some illuminate packages are use in this package. The laravel/framework doesn't require any illuminate packages

[https://packagist.org/packages/laravel/framework](https://packagist.org/packages/laravel/framework)

**Review by lahaxearnaud — COMMENTED** [review]
hello @BoGnY,
Thanks for this  pull requests. Can you just re-add illuminate packages in the composer.json ?

Arnaud

File: composer.json (hunk 1 above)
Comment (BoGnY) [reply composer.json:21]: Readded 👍 
I thought it was enough to indicate the [laravel/framework](https://github.com/laravel/framework) package, as this replaces all the illuminated packages..

**Comment by lahaxearnaud** [comment]
Merged and tagged as 1.6.0

**Comment by lahaxearnaud** [comment]
Thanks @BoGnY !

**Pull request merged**